build/README.md

```diff
@@ -129,3 +129,38 @@ Used to automate the routine operations with release branches
 #### CompleteHotfix:
 - Merges hotfix branch into master
 - Adds tag and pushes
+
+## Packages management
+#### Init
+Creates vc-package.json boilerplate with the latest version number of the platform.
+Version number can be specified by PlatformVersion parameter
+For example:
+```console
+vc-build Init
+vc-build Init -PlatformVersion 3.52.0
+```
+#### Install
+Gets the -Module parameter, which is an array of module ids, and updates vc-package.json.
+Also module id can be  supplemented with the version number.
+Discovery and Probing directories can be overrided via -DiscoveryPath and -ProbingPath
+Examples:
+```console
+vc-build install -Module VirtoCommerce.Cart VirtoCommerce.Catalog
+vc-build install -Module VirtoCommerce.Cart VirtoCommerce.Catalog:3.38.0
```

Comment:
should there be -DiscoveryPath and -ProbingPath parameters added?

Reply:
These params are for InstallModules target. I added an example there.

Reply:
I've added an extra example